### ISSUE TITLE
Issue with boro parsing for Staten Island

### ISSUE THREAD

**Issue by mendelB** [JustFixNYC/eviction-free-nyc#40]
Hi, I was at the [Contentful Meetup last night at DoSomething](https://www.meetup.com/CtlfDevsNYC/events/252935782/), where @romeboards gave an awesome presentation about the Eviction Free NYC platform, which we really enjoyed!

[@shmsh5000](https://github.com/mshmsh5000) (also at the meetup) noticed an interesting line of code in the [`utils/logic.js` file](https://github.com/JustFixNYC/eviction-free-nyc/blob/master/src/utils/logic.js#L22-#L23) wherein the `boro` variable is set to a lower-cased version of the `user.boro`:
```js
let boro = user.boro.toLowerCase();`
```
causing the conditional for `STATEN ISLAND`:
```js
if(boro === 'STATEN ISLAND') boro = 'staten';
```
 to presumabely never run, since staten island would be lower cased (`'staten island'`) at this point.

I tried testing the website flow with a Staten Island address, and when redirected to the `/guide` URL at the end of the 'steps' flow, encountered a `404`. The URL was:
`https://www.evictionfreenyc.org/en-US/guide/staten%20island/nonpayrtc?step=4`.

I tried removing the `%20island`. (presuming that was the intended boro key based on that conditional logic), changing the URL to `https://www.evictionfreenyc.org/en-us/guide/staten/nonpayrtc/?step=4` which indeed worked perfectly!

Hope this is helpful! Thanks for the amazing work that you're doing 🙌

**Comment by sraby** [comment]
@mendelB, thank you! Sorry this fell through the cracks, but it looks like Dan made a quick fix for this a long time ago: https://github.com/JustFixNYC/eviction-free-nyc/commit/a38605cc4cdd7b03cac09b5425a74ec2f495a65d.

Going to close this issue but let us know if you find anything else! Thanks again for your help.

**Comment by mendelB** [comment]
> @mendelB, thank you! Sorry this fell through the cracks, but it looks like Dan made a quick fix for this a long time ago: [a38605c](https://github.com/JustFixNYC/eviction-free-nyc/commit/a38605cc4cdd7b03cac09b5425a74ec2f495a65d).
> 
> Going to close this issue but let us know if you find anything else! Thanks again for your help.

Thanks @sraby, glad this was all fixed! 😌
